### DISCUSSION TITLE
How to hide scale label and sort series

### DISCUSSION THREAD

**Discussion by my522cn** [Live-Charts/LiveCharts2#184] [Q&A]
as the title

![Snipaste_2021-09-09_09-14-23](https://user-images.githubusercontent.com/11346219/132606006-3469ae54-af70-4ab5-babe-c55346fa5002.png)

![Snipaste_2021-09-09_09-10-59](https://user-images.githubusercontent.com/11346219/132605932-c2e9cdae-e3b2-47ad-95cc-5432cd1c4f0a.png)

**Comment by beto-rodriguez** [comment]
Try setting the Visible property to false in your X axis for the first image and in the Y axis for the second one

https://github.com/beto-rodriguez/LiveCharts2/blob/master/docs/components/1.7.axes.md#visible-property
